applications/nrf5340_audio/Kconfig.defaults

```diff
@@ -25,10 +25,12 @@ config THREAD_NAME
 	bool
 	default y
 
-# Avoid multi-build system build network core firmware
-config NCS_INCLUDE_RPMSG_CHILD_IMAGE
-	bool
-	default n
+# Empty net core image is needed either DFU enabled or not
+config NCS_SAMPLE_EMPTY_NET_CORE_CHILD_IMAGE
+	bool "Dummy Net core application"
+	default y
+	help
+		Dummy Net core child image for generating bootloader
```

Comment:
Also in here.

Reply:
I do not think we use "help" in Kconfig.defaults files in our application

Reply:
Remove help section.